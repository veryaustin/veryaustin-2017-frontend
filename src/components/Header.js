import React from 'react';
import styled from 'styled-components';
import {Link} from 'react-router';
import Wrapper from './Wrapper';
import Navigation from './Navigation';

// Main Header Component Styles
const MainHeader = styled.header`
  padding: 1em 0;
  border-bottom: 1px solid #C5C5C5;
  overflow: auto;
  zoom: 1;
`;

// Logo Link Component Styles 
const Logo = styled(Link)`
  float: left;
  margin: 0;
  color: #4A4A4A;
  font-family: 'Volte Sans Rounded';
  font-size: 20px;
  font-weight: 600;
  letter-spacing: -0.30px; 
`;

const Header = () => {
  return(
    <MainHeader>
      <Wrapper>
        <Logo to="/">Austin Lauritsen</Logo>
        <Navigation />
      </Wrapper> 
    </MainHeader>
  );
};

export default Header;