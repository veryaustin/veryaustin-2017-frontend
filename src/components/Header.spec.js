import React from 'react';
import {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';
// import {Link} from 'react-router';
import Header from './Header';


describe('Header Component', () => {
  // Component Test
  it('renders the header component', () => {
    const wrapper = shallow(<Header />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});