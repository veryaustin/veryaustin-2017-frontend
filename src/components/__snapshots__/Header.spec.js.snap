exports[`Header Component renders the header component 1`] = `
<styled.header>
  <styled.div>
    <Styled(Link)
      to="/">
      Austin Lauritsen
    </Styled(Link)>
    <Navigation />
  </styled.div>
</styled.header>
`;
